### PR TITLE
Adding fix to allow one of the either or options to be supplied.

### PR DESCRIPTION
**Pull request by crbaker**
For example, the value 'Private' could not be chosen as an option for 'ip-address', it would only recognise 'Public' as a valid option

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-name: 'Deploy to Azure Container Instances'
+name: 'WeThinkCode_ Deploy to Azure Container Instances'
```
Comment (sejo) [inline action.yml:1]: Please remove the WeThinkCode_

File: action.yml [hunk 2]
```diff
@@ -123,7 +123,7 @@ outputs:
     app-url:
         description: 'URL of the Deployed Application'
 branding:
-    icon: 'azure-logo.svg'
+    icon: 'wtc-logo.svg'
```
Comment (sejo) [inline action.yml:126]: Remove company branding and set back to azure-logo

File: lib/taskparameters.js [hunk 3]
```diff
@@ -1,270 +1,270 @@
...
+        else {
+            if (gpuCount && !gpuSku) {
+                throw Error("GPU SKU is not specified for the count. Please provide the `gpu-sku` parameter");
+            }
+            this._gpuCount = parseInt(gpuCount);
+            this._gpuSKU = (gpuSku == 'K80') ? 'K80' : (gpuSku == 'P100' ? 'P100' : 'V100');
+        }
+        this._image = core.getInput('image', { required: true });
+        let ipAddress = core.getInput('ip-address');
+        if (["Public", "Private"].indexOf(ipAddress) < 0) {
```
Comment (sejo) [inline lib/taskparameters.js:48]: why not use .includes(ipAddress) ?

**Comment by sejo** [comment]
Remove the branding if we want this merged ;)

**Comment by mauricioborges** [comment]
@sejo will create another pull request with the requested changes

**Comment by kanika1894** [comment]
@mauricioborges, If a separate PR is targeted for the required changes, are we good to close this one?

**Comment by mauricioborges** [comment]
> @mauricioborges, If a separate PR is targeted for the required changes, are we good to close this one?

Definitely @kanika1894 . I understand the PR I created #43 solves this issue in a more targeted way, what do you  think?

**Comment by kanika1894** [comment]
As required changes are handled in a separate PR, closing this.

**Pull request closed without merging**